app/scss/imports/modal.scss

```diff
@@ -0,0 +1,35 @@
+.overlay {
+  background: rgba(0,0,0,0.6);
```

Comment:
Why dont you use a scss variable instead of using the rgba color itself. We can change the styling more easily later on